src/Abp/Domain/Values/DiffIgnoreAttribute.cs

```diff
@@ -0,0 +1,8 @@
+﻿using System;
+
+namespace Abp.Domain.Values
+{
+    public class DiffIgnoreAttribute : Attribute
```

Comment:
What about renaming to `IgnoreOnCompareAttribute`?

Reply:
Roger!